lib/ui/splash/login_screen.dart

```diff
@@ -0,0 +1,228 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+
+import '../../gen/assets.gen.dart';
+import '../../theme/app_colors.dart';
+import '../widget/dimmed_image_background.dart';
+import '../widget/primary_text_form_field_decoration.dart';
+
+const _logoRevealDuration = Duration(milliseconds: 500);
+const _logoDuration = Duration(milliseconds: 750);
+const _loginFormRevealDuration = Duration(milliseconds: 700);
+
+final _shouldAnimateLogoPositionProvider =
+    StateProvider.autoDispose<bool>((_) => false);
+
+class LoginScreen extends ConsumerStatefulWidget {
```

Comment:
Please update the sub folder as well, it's `splash` currently 🙏

Reply:
Updated in https://github.com/nimblehq/ic-flutter-avishek/pull/33/commits/013d5f49facd12f481667505c58cc29314cd7e3d